openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
+    """
+    This is where the information about the site's theme gets stored to the db.
+
+    `site` field is foreignkey to django Site model
+    `theme_dir_name` contains directory name having Site's theme
+    """
+    site = models.ForeignKey(Site, related_name='themes')
```

Comment:
The idea is that you might have a 'stack' of themes you want to apply, which could be layered on top of each other to fill in certain overrides without having to duplicate across single themes.  And yes there would have to be some sort of z-index value (order, depth, rank, etc.) to describe the hierarchy.  However we don't want to jump into that without taking some time to think about the data model further, so we're only including the ForeignKey field spec for now.